docs/messaging/usage/index.md

```diff
@@ -70,8 +70,10 @@ On Android, you do not need to request user permission. This method can still be
 
 ## Receiving messages
 
-FCM messages can be sent to devices via a number of methods (see below). A message is simply a payload of data which can
-be used however you see fit within your application. Common use-cases for handling messages could be:
+FCM messages can be sent to *real* devices (iOS simulators do *not* handle cloud messages, in particular) via a number of methods (see below).
```

Comment:
```suggestion
FCM messages can be sent to *real* Android/iOS devices and Android emulators (iOS simulators however do *not* handle cloud messages) via a number of methods (see below).
```

Reply:
Thoughts on tweaking slightly? 😅

Reply:
Yes, that is more precise. Word-smithery always goes better with a bit of back and forth, I value it